package-lock.json

```diff
@@ -10,8 +10,8 @@
       "integrity": "sha512-bPHp6Ji8b41szTOcaP63VlnbbO5Ny6dwAATtY6JTjh5N2OLrb5Qk/Th5cRkRQhkWCt+EJsYrNB0MiL+Gpn6e3g==",
       "dev": true,
       "requires": {
-        "call-me-maybe": "1.0.1",
-        "glob-to-regexp": "0.3.0"
+        "call-me-maybe": "^1.0.1",
```

Comment:
Do we understand what causes these changes? What version of npm are you running?

Reply:
I was looking at the most recent commits by themselves. I see now that the PR overall doesn't introduce these changes.

Reply:
The changes from pinned to `^`? No idea. 
```
$ npm --version
6.4.1
```